src/fhirtypes/ElementDefinition.ts

```diff
@@ -1913,6 +1913,13 @@ export class ElementDefinition {
 
   private checkXhtml(value: string): boolean {
     try {
+      if (this.path.endsWith('div')) {
```

Comment:
To be a little more careful here, it might be best to check that the path ends with `.div` in order to avoid checking an element where the last part of the path is more than just `div`.

Reply:
That makes sense - made the change.